src/MiniCover/HitServices/MinimalHitService.cs

```diff
@@ -1,146 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-
-namespace MiniCover.HitServices
-{
-    public static class MinimalHitService
-    {
-        private static readonly object lockObject = new object();
-        private static readonly Dictionary<string, HitsTracer> files = new Dictionary<string, HitsTracer>();
-
-        public static void Init(string fileName)
-        {
-            lock (lockObject)
-            {
-
-                if (!files.ContainsKey(fileName))
-                {
-                    var fileInfo = new FileInfo(fileName);
-                    var tracer = files[fileName] = new HitsTracer();
-
-                    using (var streamReader = new StreamReader(fileInfo.Open(FileMode.OpenOrCreate, FileAccess.Read)))
```

Comment:
There is no need to read the file, during save we can just append new hits to the same file.
I think the hits file might get big, and reading and parsing it on every test execution seems a bit heavy.

Reply:
And it was only in the context of minimalHitService, if we are going to only one HitService there is no need